manifests/client.pp

```diff
@@ -1,8 +1,9 @@
 class amanda::client (
-  $remote_user      = undef,
-  $server           = "backup.${facts['networking']['domain']}",
-  $xinetd           = true,
-  $export_host_keys = false,
+  String           $ensure           = 'present',
+  Optional[String] $remote_user      = undef,
+  String           $server           = "backup.${facts['networking']['domain']}",
```

Comment:
Prefer `String[1]` to `String` to indicate that an empty string is not acceptable.

Reply:
Okay, I changed the type to a String[1].